cpp/util/etcd.cc

```diff
@@ -573,9 +573,8 @@ void EtcdClient::WatchRequestDone(WatchState* state, GetResponse* get_resp,
     return;
   }
 
-  // This is probably due to a timeout, just retry.
-  if (!child_task->status().ok()) {
-    LOG_EVERY_N(INFO, 10) << "Watch request failed: " << child_task->status();
+  if (child_task->status().error_code() == util::error::DEADLINE_EXCEEDED) {
+    VLOG(1) << "Watch request timedout: " << child_task->status();
```

Comment:
Shouldn't we restart other kinds of errors too? Or can *no* other error get here, somehow?